workers/unity/Packages/com.improbable.gdk.core/Config/ConnectionConfig.cs

```diff
@@ -5,14 +5,52 @@ namespace Improbable.Gdk.Core
 {
     public abstract class ConnectionConfig
     {
+        /// <summary>
+        ///     The type of networking that should be used.
+        /// </summary>
+        /// <remarks>
+        ///    Default is RakNet.
+        /// </remarks>
         public NetworkConnectionType LinkProtocol = NetworkConnectionType.RakNet;
+
+        /// <summary>
+        ///     Denotes whether protocol logging should be enabled.
+        /// </summary>
+        /// <remarks>
+        ///    Default is false.
+        /// </remarks>
         public bool EnableProtocolLoggingAtStartup = false;
+
+        /// <summary>
+        ///     Denotes whether the worker uses the local IP to connect to the SpatiaLOS runtime.
+        /// </summary>
+        /// <remarks>
+        ///    Default is false.
+        ///    This should be true if connecting client-workers to cloud deployments or if connecting via the
```

Comment:
Don't need to use the word flow. In fact the first part of the sentence includes the information in the second.
Actual cases are - worker connects to a cloud deployment using the locator or via spatial connet.